### DISCUSSION TITLE
Help radicale calendar with Zimbra external calendar integation

### DISCUSSION THREAD

**Discussion by EdG91** [Kozea/Radicale#1599] [Q&A general]
Hello,
I would like to know if it is possible to integrat a caldav calendar (hosted on a radicale server) with zimbra (open source) as external calendar ?
I tried many time to integrate it in zimbra without success.
Zimbra ask for a email adress, a password, and then the url to the external caldav calendar.

May you help me ?
Best Regards,
EdG

**Comment by Draky50110** [comment]
2025 here.
Same problem : Zimbra asks for an email as username for CalDAV calendar.
Found a solution : I created an user in Radicale "users" file with an email as username and it works with Zimbra now.

Not sure this solution works with LDAP integration though...
